### ISSUE TITLE
`comment-is-too-wide` lint and corresponding documentation mismatch in character limit

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1269]
## Problem

The `comment-is-too-wide` lint enforces a width of **100 characters**, but its motive documents a limit of **80 characters**.

## Details

`src/main/resources/org/eolang/lints/comments/comment-is-too-wide.xsl:11`:

```xsl
<xsl:variable name="max" select="100"/>
```

But `src/main/resources/org/eolang/motives/comments/comment-is-too-wide.md` says:

> Each comment should not be wider than 80 characters.

and its incorrect example is "a very long comment that contains more than eighty characters and should be flagged by the lint as too wide."

## Impact

- Code and documentation contradict each other: a comment of 90 characters is flagged by the lint but is fine according to the motive, and vice versa.
- Anyone relying on the motive to know the rule gets wrong guidance.
- The same confusion affects `comment-without-dot`/`comment-not-capitalized` docs only if they mention a width — this one explicitly does.

## Expected behaviour

Make the code and the motive agree. Either:
- change `$max` to `80` to match the documented rule, or
- update the motive (and its example) to say `100` if 100 is the intended limit.

A pack test asserting the actual threshold would prevent future drift.

**Comment by VasilevNStas** [comment]
@volodya-lombrozo @yegor256 
I want to offer you what I’ve found. Thank you in advance

**Comment by yegor256** [comment]
Fixed in #1293, merged as 64eeafd.

The contradiction was real, and the docs were the stale side: commit 11d6464 ("100 chars per line in comments", 2025-01-22) deliberately raised `$max` from 80 to 100 in the XSL and left the motive at 80. So the motive now documents 100 characters, and two boundary packs pin the actual threshold (100 characters allowed, 101 flagged, with the defect message asserted) so the code and the docs cannot drift apart silently again.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+306](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).

**Comment by 0crat** [comment]
@yegor256 Thanks for closing this issue! You've earned +12 points for this: +8 as a basis; +4 for resolving it in less than 12 hours. Your running score is [+2129](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
